zentral/contrib/santa/serializers.py

```diff
@@ -63,6 +62,35 @@ def update(self, instance, validated_data):
         return super().update(instance, validated_data)
 
 
+# Targets
+
+class TargetFilesSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = File
+        exclude = ('mt_hash', 'mt_created_at')
```

Comment:
You can exclude `id`, `source`, `signed_by` too.

Reply:
implemented